enzyme/test/Enzyme/ForwardMode/badcallused.ll

```diff
@@ -44,31 +44,34 @@ attributes #1 = { noinline nounwind uwtable }
 
 ; CHECK: define internal {{(dso_local )?}}void @diffef(double* nocapture %x, double* nocapture %"x'")
 ; CHECK-NEXT: entry:
-; CHECK-NEXT:   %call = tail call zeroext i1 @subf(double* %x)
-; CHECK-NEXT:   call void @diffesubf(double* %x, double* %"x'")
-; CHECK-NEXT:   %sel = select i1 %call, double 2.000000e+00, double 3.000000e+00
+; CHECK-NEXT:   %0 = call { i1 } @diffesubf(double* %x, double* %"x'")
```

Comment:
Super minor point, but could we name the forward diff functions with a different prefix?